src/transformState.js

```diff
@@ -5,7 +5,32 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  const changedState = state;
+
+  for (const action of Object.values(actions)) {
```

Comment:
can be simplified to
```suggestion
  for (const action of actions) {
```

Reply:
Thank you 😊